src/components/core/geoChart/GeoChartLegendRenderer.tsx

```diff
@@ -23,6 +22,7 @@ export interface IGeoChartLegendRendererProps {
     position?: string;
     responsive?: boolean;
     showFluidLegend?: boolean;
+    numericSymbols: string[];
```

Comment:
Can we mark this as `optional` to avoid using `Partial`?
At the time we access it, we can assign it default numericSymbols

Reply:
It can avoid the `Partial`. But i think required property is more important. And `numericSymbols` property always be set for `GeoChartLegendRenderer`  where it is called.
What do you think?